hack/ovn-kind-cno.sh

```diff
@@ -149,6 +158,12 @@ if [ "$BUILD_OVN" = true ] || [ "$BUILD_CNO" = true ]; then
   popd
 fi
 
+# CNO needs to get scheduled on first node, because thats where KIND will put /etc/kubernetes/kubeconfig
+until kubectl get pod -n openshift-network-operator -o wide | grep "ovn-control-plane ";do
```

Comment:
exit the script if this operation times out after a few retries?

Reply:
yeah I can add that to avoid some unforeseen infinite loop